lte/gateway/c/oai/tasks/s6a/s6a_task.c

```diff
@@ -85,8 +85,12 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
         OAILOG_DEBUG(
             LOG_S6A, "Sending s6a AIR for imsi=%s\n",
             received_message_p->ittiMsg.s6a_auth_info_req.imsi);
+        OAILOG_DEBUG(
+            LOG_S6A, "State PLMN " PLMN_FMT "from proto",
+            PLMN_ARG(
+                &received_message_p->ittiMsg.s6a_auth_info_req.visited_plmn));
       } else {
-        OAILOG_ERROR(
+        OAILOG_DEBUG(
```

Comment:
Revert this, it is an Error if the AIR cannot be sent

Reply:
Done